src/compose/volume.ts

```diff
@@ -148,6 +152,14 @@ export class Volume {
 			name: match[2],
 		};
 	}
+
+	private static omitSupervisorLabels(labels: LabelObject): LabelObject {
+		// TODO: Export these to a constant
```

Comment:
What does this todo item mean?

Reply:
We shouldn't really be hardcoding any strings if we can help it, other than to add them to the constants module. That being said, there's a bunch of them all over the codebase so I'd rather handle them at once.